Sources/StreamChatTestTools/Controllers/ChatChannelController_Mock.swift

```diff
@@ -16,8 +16,8 @@ public class ChatChannelController_Mock<ExtraData: ExtraDataTypes>: _ChatChannel
         channel_mock ?? super.channel
     }
     
-    public private(set) var messages_mock: [_ChatMessage<ExtraData>]?
-    override public var messages: [_ChatMessage<ExtraData>] {
+    public private(set) var messages_mock: LazyCachedMapCollection<_ChatMessage<ExtraData>>?
+    override public var messages: LazyCachedMapCollection<_ChatMessage<ExtraData>> {
         messages_mock ?? super.messages
     }
```

Comment:
This is not the ideal solution really. `messages_mock` should just be `[_ChatMessage<ExtraData>]` and then you can do:
```swift
message_mock.map { $0.lazyCachedMap { $0 } } ?? super.messages
```
Look at other mocks how it's done there.